.circleci/config.yml

```diff
@@ -268,9 +269,31 @@ jobs:
               $CODECOV_CMD || (sleep 5 && $CODECOV_CMD) || (sleep 5 && $CODECOV_CMD)
       - store_artifacts:
           path: glean-core/android/build/reports/tests
+      - run:
+          name: Install packaging dependencies
+          command: |
+            sudo apt update
+            sudo apt install -y gcc-mingw-w64
```

Comment:
Every time I see this I wish one could do something similar for Apple targets...